checks/libs/win/winpdh.py

```diff
@@ -0,0 +1,49 @@
+import time
+import sys
+import win32pdh
+import psutil
+
+class WinPDHCounter(object):
+    
+    def __init__(self, class_name, instance_name):
+        self._class_name = class_name
+        self._instance_name = instance_name
+
+    def get_single_value(self):
+        hq = win32pdh.OpenQuery()
+        path = win32pdh.MakeCounterPath((None, self._class_name, None, None, 0, self._instance_name))
+        counter_handle = win32pdh.AddCounter(hq, path)
+        win32pdh.CollectQueryData(hq)
```

Comment:
From what i understand `hq` is stateful

> Some performance counters are instantaneous. Others require you collect data for a few seconds. See references for details. A safe bet is to collect data twice before getting values.
 
(https://www.cac.cornell.edu/wiki/index.php?title=Performance_Data_Helper_in_Python_with_win32pdh)


Shall it be kept in the instance to persist over time?